source/config.js

```diff
@@ -11,6 +11,10 @@ var defaultConfig = {
 
   // The base URL ungit will be accessible from.
   urlBase: "http://localhost",
+  
+  // The URL root path under which ungit will be accesible.
+  // Either blank or starting with a slash and no trailing slash
```

Comment:
I'm just saying that, from a UX perspective, it would be nice if ungit automatically appended a slash to the start of the config variable if needed, and removed any trailing slash. Just one less thing for the user to worry about.

Alternately, if taking care of this constraint automatically would take too much development effort, that requirement for the formatting of that setting should definitely be documented somewhere other than just a comment in the source code.